### ISSUE TITLE
What is the performance on GPT4?

### ISSUE THREAD

**Issue by Zui-C** [zhao-zilong/ssc-cot#1]
Hi, I am very interested in your method.
But I noticed that Table 1. Result on MATH.  shows 27.4 in total.  And it's based on LLEMMA.
I would like to know if you have tried it on GPT4 or GPT3.5. Can it exceed 51.8% like using PAL on GPT-4?

**Comment by zhao-zilong** [comment]
Hi @Zui-C 

Actually our results are based on gpt 3.5. 
On Math, we did not run on GPT4, but for TriMaster100, I indeed tested on GPT4, but not for 20 runs per questions. I got 89/750 points. Given that GPT3.5 only scores half of that on TriMaster100, SSC-CoT with GPT4 can indeed increases its performance, but it is too slow and expensive, we end this experiment after several tries. 

Best,

Zilong
